cypress/e2e/marc/marc-bibliographic/edit-marc-bib/edit-marc-bib-add-multiple-001s.cy.js

```diff
@@ -0,0 +1,101 @@
+import Permissions from '../../../../support/dictionary/permissions';
+import DataImport from '../../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import TopMenu from '../../../../support/fragments/topMenu';
+import Users from '../../../../support/fragments/users/users';
+import QuickMarcEditor from '../../../../support/fragments/quickMarcEditor';
+import { INSTANCE_SOURCE_NAMES } from '../../../../support/constants';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+
+describe('MARC -> MARC Bibliographic -> Edit MARC bib', () => {
+  const testData = {
+    tag001: '001',
+    tag001Content: '$a n 94000339',
+    tag001ValueInSourceMask: /[a-z]+\d+/,
+  };
+
+  const marcFiles = [
+    {
+      marc: 'oneMarcBib.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      numOfRecords: 1,
+    },
+  ];
+
+  const createdAuthorityIDs = [];
+
+  before('Create test data', () => {
+    cy.createTempUser([
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorAll.gui,
+    ]).then((createdUserProperties) => {
+      testData.userProperties = createdUserProperties;
+
+      marcFiles.forEach((marcFile) => {
+        cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(
+          () => {
+            DataImport.verifyUploadState();
+            DataImport.uploadFileAndRetry(marcFile.marc, marcFile.fileName);
+            JobProfiles.waitLoadingList();
+            JobProfiles.search(marcFile.jobProfileToRun);
+            JobProfiles.runImportFile();
+            JobProfiles.waitFileIsImported(marcFile.fileName);
+            Logs.checkStatusOfJobProfile('Completed');
+            Logs.openFileDetails(marcFile.fileName);
+            for (let i = 0; i < marcFile.numOfRecords; i++) {
```

Comment:
I think we don't need a loop here if it is doing only one iteration. Could you please remove for loop?